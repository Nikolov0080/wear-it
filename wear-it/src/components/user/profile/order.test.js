import React from 'react';
import renderer from 'react-test-renderer';
import Order from './order';
import { BrowserRouter } from 'react-router-dom';

it('renders correctly Order', () => {
    const tree = renderer
        .create(<BrowserRouter>
            <Order data={{
                imageURL: "fkjgsdh",
                price: "45",
                productName: "meow",
                size: 'XXXXXXXlsd'
            }
            }

            />
        </BrowserRouter>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});