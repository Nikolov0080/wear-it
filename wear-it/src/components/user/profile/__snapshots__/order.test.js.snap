// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly Order 1`] = `
<div>
  <div
    className="orderBox"
  >
    <img
      alt="product"
      className="image"
      src="fkjgsdh"
    />
    <div
      className="productName"
    >
      Product name:  
      meow
    </div>
    <div
      className="size"
    >
      Size: 
      XXXXXXXlsd
    </div>
    <div
      className="price"
    >
      Price: 
      45
       USD
    </div>
  </div>
</div>
`;
